substratevm/src/com.oracle.objectfile.jdk11/src/com/oracle/objectfile/ModuleAccess.java

```diff
@@ -31,11 +31,18 @@
     public static void openModuleByClass(Class<?> declaringClass, Class<?> accessingClass) {
```

Comment:
Seems that there is exactly the same method defined in `ModuleSupport`. Could it be unified?

Reply:
Nope. Because the objectfile distribution does not depend on the svm distribution and the objectfile distribution also does not depend on the graal distribubtion. Thus there is no place where we could put this stuff that allow reuse from both sides.

Reply:
Okay, I see.